app/controllers/users_controller.rb

```diff
@@ -23,10 +24,51 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update(user_params)
+      flash[:success] = t "profile_updated"
+      redirect_to @user
+    else
+      flash[:success] = t "profile_updated_error"
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user && @user.destroy
```

Comment:
```suggestion
    if @user.destroy
```
k cần check @user vì nếu k có user thì sẽ k vào đến đây được